build/storage/python_system_tools/test_platform.py

```diff
@@ -0,0 +1,262 @@
+import logging
+import re
+from concurrent.futures import ThreadPoolExecutor
+from typing import Optional
+
+from python_system_tools.terminal import SSHTerminal
+from python_system_tools.consts import WORKSPACE_PATH, PMS
+
+
+class UnknownOSError(Exception):
```

Comment:
unknown is bad name because custom error should specify error and help find a bug in the future, if you don't know error you can explicity use Exception but it's not proper - you should know what kind error can be raised

Reply:
It was supposed to mean that current operating system is unknown, not that the error is unknown. I changed it to UnsupportedOSError.